test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -466,14 +465,13 @@ private static IntHashSet asSet(int[] array)
         return set;
     }
 
-    public interface SchemaSpec
+    public interface TestState
```

Comment:
why this refactor?  This isn't the `TestState` and is confusing as we have `State`... please revert, it makes the diff larger, and adds confusion.  I am fine with `table()` and `keyspace()` if you wish to make that change, but please leave it `SchemaSpec` as this is *only* schema